packages/forms/src/components/ui/text-input/StandardTextInput.tsx

```diff
@@ -170,14 +171,14 @@ export const StandardTextInput: React.FC<StandardTextInputProps> = ({
   const { colors } = useThemeFields(
     {
       colors: {
-        disabledBackgroundColor: theme.disabledBackgroundColor,
+        disagledBackgroundColor: theme.backgroundColorDisabled,
```

Comment:
No need to resolve these from the theme. You can just pass them to `Box` and it will resolve them.
The following are resolved from theme in Box:
* background
* color
* border
* borderColor